cardano-cli/src/Cardano/CLI/EraBased/Options/Common.hs

```diff
@@ -1566,16 +1566,21 @@ pPoolIdOutputFormat =
     , Opt.value IdOutputFormatBech32
     ]
 
-pTxViewOutputFormat :: Parser TxViewOutputFormat
-pTxViewOutputFormat =
-  Opt.option readTxViewOutputFormat $ mconcat
+pTxViewOutputFormat :: Parser ViewOutputFormat
+pTxViewOutputFormat = pViewOutputFormat "transaction"
+
+-- | @pViewOutputFormat kind@ is a parser to specify in which format
+-- to view some data (json or yaml). @what@ is the kind of data considered.
+pViewOutputFormat :: String -> Parser ViewOutputFormat
+pViewOutputFormat kind =
+  Opt.option readViewOutputFormat $ mconcat
     [ Opt.long "output-format"
     , Opt.metavar "STRING"
     , Opt.help $ mconcat
-      [ "Optional transaction view output format. Accepted output formats are \"json\" "
+      [ "Optional " <> kind <> " view output format. Accepted output formats are \"json\" "
```

Comment:
you can use a comma instead of `<>`

Reply:
Ah true that, because of the leading `mconcat`. Good catch :+1:

Change done